package/com.unity.formats.usd/Documentation~/index.md

```diff
@@ -40,7 +40,10 @@ Once the USD package is installed, a USD menu will appear, enabling you to easil
 
 ## Package Limitations
 
-* The USD plugin bundles are not currently code signed, so will need to be manually signed or removed from standalone player builds.
+* The USD plugin bundles are not currently code signed, so will need to be manually signed. 
+    * In some cases on MacOS devices, this may mean manually removing the 'com.apple.quarantine' meta flag from the UsdCs.bundle plugin, eg: run `xattr -d com.apple.quarantine <UsdCs.bundle path>`.
```

Comment:
Should we indicate to users where in the package folder subtree the bundle lives?

Reply:
Done :)